user_docs/en/userGuide.t2t

```diff
@@ -2100,6 +2100,71 @@ Please see the [EcoBraille documentation ftp://ftp.once.es/pub/utt/bibliotecnia/
 | Toggle braille tethered to | A |
 %kc:endInclude
 
+++ BP12 and EL12 series ++
+The BraillePen 12 and BraillePen 12 Touch from [Harpo http://int.harpo.com.pl/] also EasyLink 12 and EasyLink 12 Touch from Optelec are supported when connected via Bluetooth.
+
+Following are the key assignments for this display with NVDA.
+Please see the display's documentation for descriptions of where these keys can be found.
+
+%kc:beginInclude
+|| Name | Key |
+| Scroll braille display back | left scroll |
+| Scroll braille display forward | right scroll |
+| Route to braille cell | routing keys |
+| upArrow key | joystick up |
+| downArrow key | joystick down |
+| leftArrow key | joystick left |
+| rightArrow key | joystick right |
+| enter key | space+dot8 |
+| backspace key | space+dot7 |
+| delete key | space+dot1+dot4+dot5 |
+| windows key | space+dot3+dot4 |
+| alt key | space+dot1+dot3+dot4 |
+| escape key | space+dot1+dot5 |
+| Move braille display to previous line | joystick up+space |
+| Move braille display to next line | joystick down+space |
+| shift+tab key | space+joystick left |
+| tab key | space+joystick right |
+| Report next word in review | space+dot5 |
+| Report previous word in review | space+dot2 |
+| Report next line in review | space+dot4 |
+| Report previous line in review | space+dot1 |
+| Report next character in review | space+dot6 |
+| Report previous character in review | space+dot3 |

```

Comment:
nit: For consistency, these should be "Move to next word in review", etc., rather than "Report next word in review", etc.